functional-tests/lib/app.js

```diff
@@ -8,6 +8,7 @@ const bodyParser = require('body-parser');
 const partials = require('express-partial-templates');
 const template = require('hof-govuk-template');
 const cookieParser = require('cookie-parser');
+const mockPostcode = require('../../mock-postcode');
```

Comment:
Yes please.

I'd also add the whole test directory to a `.npmignore` file to keep the installed package light.